lib/logstash/filters/kubernetes_metadata.rb

```diff
@@ -0,0 +1,223 @@
+# encoding: utf-8
+require "logstash/filters/base"
+require "logstash/namespace"
+require "lru_redux"
+require "rest-client"
+require "uri"
+require "logstash/json"
+
+# Process the metadata include in the file names of kubernetes log files
```

Comment:
Updated with simple documentation on how to use and configure the plugin. I can go into more detail but this should be enough to get us going.